insQ.js

```diff
@@ -10,7 +10,7 @@ var insertionQ = (function () {
         elm = document.createElement('div'),
         options = {
             strictlyNew: true,
-            timeout: 20
+            timeout: 0
```

Comment:
If the reason for this is that the configuration is not actually being passed through we may as well remove the ability to configure and set strictlyNew = false, timeout: 0, always (really therefore remove strictlyNew and timeout and the code related to those options) @cloudysail @AliTajeldin what do you think?

Reply:
It is because we are setting a flag at the start of the definition.  So we end up latching the default option value.  We should check the actual option value when we are ready to register the callback (inside `listen`)